apps/server/src/graphql/resolvers.ts

```
@@ -0,0 +1,1239 @@
+          () => ({ startDate: startDateStr, endDate: endDateStr, pageSize }),
+          logger
+        );
+
+        const allProtocolMetrics = await queryAllSubgraphsWithPerChainVariables<SingleChainProtocolMetricsResponse>(
+          PROTOCOL_METRICS_DATE_RANGE,
+          () => ({ startDate: startDateStr, endDate: endDateStr, pageSize }),
+          logger
+        );
+
+        // Track successful/failed chains
+        allTokenRecords.successfulChains.forEach(c => successfulChains.add(c));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)